src/main/java/org/kuali/kfs/sys/document/workflow/FinancialSystemSearchableAttribute.java

```diff
@@ -133,51 +134,52 @@ protected List<Row> getSearchingRows(String documentTypeName) {
                     }
                 }
 
-                BusinessObject alBusinessObject;
+                final BusinessObject alBusinessObject;
                 try {
                     alBusinessObject = (BusinessObject) alClass.newInstance();
-                } catch (Exception cnfe) {
+                } catch (final Exception cnfe) {
                     throw new RuntimeException("Unable to instantiate accounting line class: " + alClass, cnfe);
                 }
 
-                Field chartField = FieldUtils.getPropertyField(alClass, KFSPropertyConstants.CHART_OF_ACCOUNTS_CODE,
+                final Field chartField = FieldUtils.getPropertyField(alClass, KFSPropertyConstants.CHART_OF_ACCOUNTS_CODE,
                         true);
                 chartField.setFieldDataType(CoreConstants.DATA_TYPE_STRING);
-                chartField.setColumnVisible(true);
+                chartField.setColumnVisible(false);
```

Comment:
It looks like we had previously set these various `columnVisible` properties to true for some reason (or we somehow missed a previous KualiCo change for setting them to false). Do the doc-type-specific document searches still behave as before if this property is set to false?

Reply:
updated